### DISCUSSION TITLE
Snippets and emets

### DISCUSSION THREAD

**Discussion by abuhabban** [coder/code-server#4012] [General] [answered]
I want to know where the all built-in snippet and Emets stored. 

> I want to see all built-in emets and snippet. But i couldn't find because there where many file and folder try to help me.

**Comment by lg-nineoneninethreenine** [comment]
Can you please explain about this? I am not able to understand.

**Comment by jsjoeio** [comment, marked as answer]
Here's how you can figure that out:

1. run code-server
2. open Command Palette
3. select "Preferences: Configure User Snippets"
4. hit "New Global Snippets file" and type name
5. right click file tab and select "Copy Path"

For me, it's `/Users/jp/.local/share/code-server/User/snippets/hello.code-snippets`

So to answer your question, snippets are stored under `.local/share/code-server/User/snippets`
